.history/README_20230324183049.md

```diff
@@ -0,0 +1,4 @@
+# Personal-portfolio-microverse
+This project is my personal portfolio development at the start of Microverse.
+
+Microverse already prepared template in figma for all students to replicate. This is also a milestone project so not everything was completed at a go.
```

Comment:
- [ ] Good description, but this is not how the readme should be presented, at microverse we use a custom readme template file to follow, please make sure you check over this [link](https://github.com/microverseinc/readme-template) and update your readme file referring to this template, it is mandatory to use it.

<img width="916" alt="2" src="https://user-images.githubusercontent.com/77843050/227921629-9e9ba884-f343-4b26-bb03-800729a0c71e.png">